spec/editors_draft.html

```diff
@@ -3298,6 +3298,261 @@ <h2>Evaluation Semantics</h2>
+      </p>
+      <pre id="load-renaming-blank-example-simple"
+         title="Loading of a cdt:List literal with a blank node"
+         data-transform="updateExample"
+         data-content-type="text/turtle"
+         class="nohighlight example">
+      <!--
+      ex:s ex:p1 "[ _:b0 ]"^^cdt:List .
+      -->
+      </pre>
+      <pre id="load-renaming-blank-example-simple-result"
+         title="Loading of a cdt:List literal with a blank node: Result"
+         data-transform="updateExample"
+         data-content-type="text/n-triples"
+         class="nohighlight example">
+      <!--
+      <http://example.org/s> <http://example.org/p1> "[ _:b001 ]"^^<http://w3id.org/awslabs/neptune/SPARQL-CDTs/List> .
+      -->
+      </pre>
+
+      <p>
+        Evaluating a SPARQL Update in <a href="#load-renaming-blank-example-load-twice"></a> with two `INSERT DATA` operations with the same blank-node-containing <a>cdt:List literal</a> MUST result in a system-internal representation of two triples whose objects are <a>cdt:List literals</a> containing different blank nodes that, in the lexical form of these literals, are captured by different blank node identifiers (e.g., <a href="#load-renaming-blank-example-load-twice-result"></a>).
+      </p>
+      <pre id="load-renaming-blank-example-load-twice"
+         title="Repeated loading of cdt:List literals with a blank node"
+         data-transform="updateExample"
+         data-content-type="application/sparql-update"
+         class="nohighlight example">
+      <!--
+      INSERT DATA { <s> <p2> "[ _:b0 ]"^^cdt:List } ;
+      INSERT DATA { <s> <p2> "[ _:b0 ]"^^cdt:List } ;
+      -->
+      </pre>
+      <pre id="load-renaming-blank-example-load-twice-result"
+         title="Repeated loading of cdt:List literals with a blank node: Result"
+         data-transform="updateExample"
+         data-content-type="text/n-triples"
+         class="nohighlight example">
+      <!--
+      <http://example.org/s> <http://example.org/p2> "[ _:b002 ]"^^<http://w3id.org/awslabs/neptune/SPARQL-CDTs/List> .
+      <http://example.org/s> <http://example.org/p2> "[ _:b003 ]"^^<http://w3id.org/awslabs/neptune/SPARQL-CDTs/List> .
+      -->
+      </pre>
+
+      <p>
+        Loading the Turtle in <a href="#load-renaming-blank-example-consistent-ids"></a> with a triple with `_:b0` appearing in both the subject position and inside a <a>cdt:List literal</a> object MUST result in a system-internal representation with a single triple whose subject is the same blank node as contained in the single-element <a>cdt:List literal</a> object, and where that blank node MUST NOT have the identifier "b0" (e.g., <a href="#load-renaming-blank-example-consistent-ids-result"></a>).
```

Comment:
Same comment as above, I don't see how the last part of this sentence (i.e., the bnode identifier must not be "b0"), is a consequence of the import requirements that we currently have.

Reply:
Agreed. Will drop that clause.

Reply:
And another one.
```suggestion
        Loading the Turtle in <a href="#load-renaming-blank-example-consistent-ids"></a> with a triple with `_:b0` appearing in both the subject position and inside a <a>cdt:List literal</a> object MUST result in a system-internal representation with a single triple whose subject is the same blank node as contained in the single-element <a>cdt:List literal</a> object (e.g., <a href="#load-renaming-blank-example-consistent-ids-result"></a>).
```